designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleStateStatus.scala

```diff
@@ -8,14 +9,16 @@ import pl.touk.nussknacker.engine.deployment.DeploymentId
 
 import java.net.URI
 
-object SimpleStateStatus {
+object SimpleStateStatus extends LazyLogging {
 
   def fromDeploymentStatus(deploymentStatus: DeploymentStatus): StateStatus = {
     deploymentStatus match {
       case noAttributes: NoAttributesDeploymentStatus => NoAttributesStateStatus(noAttributes.name.value)
       // We assume that all deployment status have default allowedActions. Non-default allowedActions have only
       // statuses that are not deployment statuses but scenario statuses.
-      case problem: ProblemDeploymentStatus => ProblemStateStatus(problem.description)
+      case problem: ProblemDeploymentStatus =>
+        logger.warn(s"Problem deployment status: ${problem.description}")
```

Comment:
logging warn in factory method is not the best approach, it is rather service-related job. In `ScenarioStatusProvider:196` we have debug logging for scenario status, maybe we should log warn for problem statuses there?